t/t-large-files-on-windows.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+test_description='test large file handling on windows'
+. ./test-lib.sh
+
+test_expect_success SIZE_T_IS_64BIT 'require 64bit size_t' '
+
+	dd if=/dev/zero of=file bs=1M count=4100 &&
+	git config core.compression 0 &&
+	git config core.looseCompression 0 &&
+	git add file &&
+	git verify-pack -s .git/objects/pack/*.pack &&
+	git fsck --verbose --strict --full &&
+	git commit -m msg file &&
+	git log --stat &&
+	git gc &&
```

Comment:
Almost very step is error prone and some do not repeat outside of the test environment (differing big file threshold effects I think) i.e. pack vs loose object.

Reply:
Then we really need a more robust test. Otherwise a developer might be tempted to think that they fixed a flawed regression test, all the while a real regression crept in.

So please indulge me: what exactly is this regression test case supposed to test?

Reply:
This is an expansion of the 'does it perform correctly' test that Thomas B initially developed.

Each of the fsck, verify-pack, log and gc commands can barf. Plus if run manually it actually does a slightly different test (add to loose vs pack), which I didn't get to the bottom of!

In this case he had deliberately set the compression to zero so that the compressed file would exceed the limit, we also 'need' (if we are to be complete) to check for packed and loose encoding, etc etc (including breakages at various process stages, add vs commit, 

I'd agree that the testing still has a long way to go to get a smooth set of tests that satisfy all. It's all low level code issues that can pop up in many places.

It may be that we have a two phase plan. First make the code patches visible (if the code isn't working...), then make the testing acceptable (to as yet unknown/discussed/decided standards).